src/Components/ArtsyShippingEstimate.tsx

```diff
@@ -116,7 +115,10 @@ export const ArtsyShippingEstimate = ({
   return (
     <LinkButton
       tabIndex={0}
-      onClick={() => estimateWidget.open()}
+      onClick={e => {
+        e.preventDefault()
+        estimateWidget.open()
```

Comment:
Playing with it on staging now I wonder if this should work as a toggle as well - similar to how the whole section works as a toggle for shipping section. The whole toggle/untoggle thing seems to be slightly off to me now - I personally think that the whole section should start expanded and this link to estimate should go straight into the shipping section next to shipping description. But I think this works for now. We can def see if people even pay attention to it and then try to iterate from there.

Reply:
👍  I generally agree with making these tweaks after we get to test it a little bit